script.js

```diff
@@ -0,0 +1,145 @@
+// This script let's you send a mail to everybody who fills out your form.
+// What you need:
+// a) A Google Form with an E-Mail field
+// b) A Google Spreadsheet where all the answers are stored (created out of the form)
+// c) A Google Doc where you can enter the body of the mail
+// d) Upload your PDF-Attachments in GDrive
+
+
+var SPREADSHEET = '[your Spreadsheet ID (take it form the URL)]'; //TODO Add the SpreadsheetID (created in b)
+//var INFO_FLYER_YES = 'ja';
+var SHEET_INDIVIDUALLY = '[sheet name]'; //TODO use the name of the Spreadsheet name
+//var INFO_FLYER = 'Info Fyler';
+var EMAIL_ADDRESS_FIELD = 'E-Mail address'; //TODO use the name of the mail field in the FORM
+
+var EMAIL_ADDRESS = 'E-Mail address';  //TODO use the name of the column
+var EMAIL_REPLY_TO_ADDRESS = 'volunteer@keshaniya.org'; //TODO use the address where replys to your mail should end up
+var EMAIL_DISPLAY_NAME = 'Keschaniya Kitchen'; //TODO the displayed name of sender
+
+var SUBJECT = '[Autoresponse] ...'; //TODO edit as you desire
+
+var BODY = '10Cb6U3xEmMQjUIm123KAQ3p5QlNe6Z1NJim3Wpb6_Jw'; //TODO use the ID of the document created in c)
+
+
+// TODO  Use the ID of the PDF and insert it here:
+var ATTACHMENT1 = '[ID]';
+var ATTACHMENT2 = '[ID]';
+var ATTACHMENT3 = '[ID]';
+
+var FORM_EMAIL = '[ID]'; // TODO: get ID via the 'test-function'
+
+ //this section is for testing your script.
+ function test() {
+     //the ID is the same as before
+    var range = SpreadsheetApp.openById('[ID]').getSheetByName(SHEET_INDIVIDUALLY).getRange('a1');
+    var values = {
+        'E-Mail address': ['/* use your mail address */'],
+
+    }
+    var e = {
+        range: range,
+        namedValues: values
+    }
+    onFormSubmit(e);
+}
+
+// this part starts the script on form submition
+function onFormSubmit(e) {
+    var form = FormApp.getActiveForm();
+    var emailItem = form.getItemById(FORM_EMAIL);
+
+    var emailResponse = e.response.getResponseForItem(emailItem);
+
+    var email = emailResponse.getResponse();
+    var language = languageResponse.getResponse();
+    var transportation = transportationResponse.getResponse();
+
+    //
+    var packaging = DriveApp.getFileById(ATTACHMENT1);
+    var volunteerInfo = DriveApp.getFileById(ATTACHMENT2);
+    var labels = DriveApp.getFileById(ATTACHMENT3);
+
+
+    // TODO edit attachment names if needed (see ln 16)
+    var attachments = [packaging.getAs(MimeType.PDF),
+        volunteerInfo.getAs(MimeType.PDF),
+        labels.getAs(MimeType.PDF)];
+
+
+// the following part is only needed when you want to use multiple languages or mails with different content based on answers!
```

Comment:
Well indent correct indentation would be great, but I could do this, before merging.